core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -364,7 +367,7 @@ angular.module('oppia').directive('conversationSkin', [
         'PlayerTranscriptService', 'QuestionPlayerEngineService',
         'QuestionPlayerStateService', 'ReadOnlyCollectionBackendApiService',
         'RefresherExplorationConfirmationModalService',
-        'SiteAnalyticsService', 'StateCardObjectFactory',
+        'SiteAnalyticsService',
```

Comment:
Please don't just leave this here add other imports (from next lines) to this line.

Reply:
Done.